app/posts/apply-for-teacher-training/2023-10-31-blocking-candidates-with-no-visa.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Blocking candidates from applying for courses that do not sponsor visas
+description:
+date: 2023-10-31
+tags:
+  - visa sponsorship
+  - international
+---
+
+There was an agreement that it is beneficial to block certain user groups from submitting an application. Specifically when a course will not sponsor visas, and the candidate does not have the right to work or study in the UK.
+
+## The issue
+
+In 2022-2023 cycle we have had over 4,800 candidate applications rejected due to not having visa sponsorship. This wastes candidate and provider time, whilst also reducing candidates' chance of finding a course match.
+
+In 2023-2024 cycle there was a high proportion of rejection on day one due to no visa.
```

Comment:
Maybe we don't need this line as we've already mentioned rejections above? What you said above is more impressive I think.